src/backend/marsha/core/validators.py

```diff
@@ -0,0 +1,19 @@
+"""Validators for the Marsha project."""
+from datetime import datetime
+
+from django.core.exceptions import ValidationError
+
+
+def validate_date_after(date):
+    """Check date is in the future."""
+    # Converts string to date if needed
```

Comment:
Do you think we should accept date in string ? When the API creates or modifies this property, do we have a `datetime` object or a string ? If we have a `datetime` object then I think we should not accept string. Keep string conversion otherwise but we loose the timezone info here.

Reply:
Normally string to date and date to string is handled by the serializer field. We should not need to worry about that.

Reply:
ok thank you